tests/test_for_support/test_for_helpers.py

```diff
@@ -63,7 +97,62 @@ def abc_pump(base):
 
 
 @register_with(MODEL_REGISTRY)
-def proton_pump(base):
+def abc_pump_annotations(base):
+    """Provide a model with an ABC transport reaction."""
+    atp = cobra.Metabolite("atp_c", compartment="c")
+    adp = cobra.Metabolite("adp_c", compartment="c")
+    h = cobra.Metabolite("h_c", compartment="c")
+    pi = cobra.Metabolite("pi_c", compartment="c")
+    h2o = cobra.Metabolite("h2o_c", compartment="c")
+    aso_c = cobra.Metabolite("aso3_c", compartment="c")
+    aso_e = cobra.Metabolite("aso3_e", compartment="e")
+
+    atp.annotation["bigg.metabolite"] = "atp"
+    atp.annotation["biocyc"] = ["META:ATP", "META:CPD0-1634"]
+    atp.annotation["metanetx.chemical"] = "MNXM13"
+    atp.annotation["seed.compound"] = "cpd00011"
+
+    adp.annotation["bigg.metabolite"] = "adp"
+    adp.annotation["biocyc"] = ["META:ADP", "META:CPD0-1651"]
+    adp.annotation["metanetx.chemical"] = "MNXM7"
+    adp.annotation["seed.compound"] = "cpd00008"
+
+    h.annotation["bigg.metabolite"] = "h"
+    h.annotation["biocyc"] = "META:PROTON"
+    h.annotation["metanetx.chemical"] = "MNXM1"
+    h.annotation["seed.compound"] = "cpd00067"
+
```

Comment:
Same here, please simplify to only use one of each principal types of annotation (list vs single string)